Kconfig.zephyr

```diff
@@ -399,6 +399,41 @@ config BOOTLOADER_ESP_IDF
 	  inside the build folder.
 	  At flash time, the bootloader will be flashed with the zephyr image
 
+config BOOTLOADER_BOSSA
+	bool "BOSSA bootloader support"
+	select USE_DT_CODE_PARTITION
+	depends on SOC_FAMILY_SAM0
+
+	help
+	  Signifies that the target uses a BOSSA compatible bootloader.  If CDC
+	  ACM USB support is also enabled then the board will reboot into the
+	  bootloader automatically when bossac is run.
+
+config BOOTLOADER_BOSSA_DEVICE
```

Comment:
This should be `BOOTLOADER_BOSSA_DEVICE_NAME` (or `..._DEV_NAME`, which seems a little more common.

```
git grep -B1 -i 'string.*device name'
```

Fixup will also need to be squashed into the introducing commit.

Reply:
Done.  Kept DEVICE_NAME to match CDC_ACM_DEVICE_NAME